client/src/components/DisplayTable.js

```diff
@@ -92,6 +96,13 @@ export class DisplayTable extends React.Component {
       .flatMap( row => _.keys(row.search_values) )
       .uniq()
       .value();
+    
+    const total_row = _.reduce(
+      sorted_filtered_data, (totals, row) => 
+        _.mapValues(totals, (total, col_key) =>
+          total + row.sort_values[col_key]),
+      _.mapValues(total_row_config, () => 0)
+    );
```

Comment:
Might just be me, but I find this indenting hard to read (hard to scan quickly, at least, need to stop and count brackets). I think a couple more new lines helps a lot, like this.
```
const total_row = _.reduce(
  sorted_filtered_data,
  (totals, row) => _.mapValues(
    totals,
    (total, col_key) => total + row.sort_values[col_key]
  ),
  _.mapValues(total_row_config, () => 0)
);
```
Easier to match up where each function starts and ends and what's an argument to what?